src/carp/character/character.entity.ts

```diff
@@ -7,4 +7,19 @@ export default class Character {
 
   @Column()
   name: string;
+
+  @Column()
+  max_health: number;
+
+  @Column()
+  current_health: number;
+
+  @Column()
+  power: number;
+
+  @Column()
+  defense: number;
+
+  @Column()
+  position: string;
```

Comment:
As above, talk of serialize/deserialize can be adopted here, and to use a custom Vector2 type.